controls/SV-230229.rb

```diff
@@ -54,33 +51,31 @@
              Public Key Algorithm: rsaEncryption
 
     If the root ca file is not a DoD-issued certificate with a valid date and
-installed in the /etc/sssd/pki/sssd_auth_ca_db.pem location, this is a finding.
-  "
-  desc 'fix', "
-    Configure RHEL 8, for PKI-based authentication, to validate certificates by
+installed in the /etc/sssd/pki/sssd_auth_ca_db.pem location, this is a finding.'
+  desc 'fix', 'Configure RHEL 8, for PKI-based authentication, to validate certificates by
 constructing a certification path (which includes status information) to an
 accepted trust anchor.
 
     Obtain a valid copy of the DoD root CA file from the PKI CA certificate
 bundle from cyber.mil and copy the DoD_PKE_CA_chain.pem into the following file:
 
-    /etc/sssd/pki/sssd_auth_ca_db.pem
-  "
+    /etc/sssd/pki/sssd_auth_ca_db.pem'
   impact 0.5
   tag severity: 'medium'
   tag gtitle: 'SRG-OS-000066-GPOS-00034'
-  tag satisfies: %w(SRG-OS-000066-GPOS-00034 SRG-OS-000384-GPOS-00167)
+  tag satisfies: ['SRG-OS-000066-GPOS-00034', 'SRG-OS-000384-GPOS-00167']
   tag gid: 'V-230229'
   tag rid: 'SV-230229r627750_rule'
   tag stig_id: 'RHEL-08-010090'
   tag fix_id: 'F-32873r567434_fix'
   tag cci: ['CCI-000185']
   tag nist: ['IA-5 (2) (a)']
 
-  ca_file = input('root_ca_file')
-
-
-  describe x509_certificate(ca_file) do
+  root_ca_file = input('root_ca_file')
+  describe file(root_ca_file) do
+    it { should exist }
+  end
+  describe x509_certificate(root_ca_file) do
     its('issuer_cn') { should match 'CN=DoD' }
   end
```

Comment:
@wdower @em-c-rod this needs to be added to the x509 certificate resource as a properity that so we can pass in the 'child' and 'parent' files and get back a bool to indicate a 'cert is part of a chain'